packages/yourloops/pages/signup/signup-radio-label.tsx

```diff
@@ -39,10 +39,10 @@ const SignupRadioLabel: FunctionComponent<RadioLabelProps> = ({ header, body })
 
   return (
     <React.Fragment>
-      <Typography aria-label="radio-label-header" variant="h6" gutterBottom>
+      <Typography aria-label={t('radio-label-header')} variant="h6" gutterBottom>
```

Comment:
Sorry that I missed this before, but this aria-label are useless. Aria label should be used to add information to icon an such, but not for text as it is already readable.
I found this when seeing the values of these two aria label (they don't make sense). So you can simply remove them, and in your test, you can either use getByText or getByTestId (if getByText is not possible)